cara_murphy/art.html

```diff
@@ -0,0 +1,254 @@
+                    artTypeButton[i].style.display = "none";
+                }
+                pageHeader = document.getElementById("artPageHeader");
+                pageHeader.style.display = "none";
+
+
+                // Show the specific tab content
+                document.getElementById(pageName).style.display = "block";
+            }
+
+            // Get the element with id="defaultOpen" and click on it
+            document.getElementById("defaultOpen").click();</script>
+
+        <h3 class="artTypeHeader" id="digital">digital</h2>
+            <hr class="artHeaderLine">
+
+
+            <div class="dynamicGrid">
+                <div class="dynamicGridColumn">
+                    <image class="portrait" src="DigitalArtPortfolio/GreenBoyHead.JPG">
+                    </image>
+                    <image class="portrait" src="DigitalArtPortfolio/caraDongDrawing.jpg">
+                    </image>
+                    <image class="portrait" src="DigitalArtPortfolio/HalfBodyDress.jpg">
+                    </image>
+                </div>
+                <div class="dynamicGridColumn">
+                    <image class="portrait" src="DigitalArtPortfolio/Rumination.PNG">
+                    </image>
+                    <image class="portrait" src="DigitalArtPortfolio/LightStudyBoy.jpg">
+                    </image>
+                    <image class="portrait" src="DigitalArtPortfolio/meBostonCommonDrawing.jpg">
+                    </image>
+                    <image class="portrait" src="DigitalArtPortfolio/YellowBoyHead.JPG">
+                    </image>
+                </div>
+
+                <div class="dynamicGridColumn">
+                    <image class="portrait" src="DigitalArtPortfolio/PromDressPlants.JPG">
+                    </image>
+                    <image class="portrait" src="DigitalArtPortfolio/IllinoisBoy.JPG">
+                    </image>
+                    <image class="portrait" src="DigitalArtPortfolio/GoldenGirl.jpg">
+                    </image>
+                </div>
+            </div>
+
+            <h3 class="artTypeHeader" id="digital">graphic art</h2>
+                <hr class="artHeaderLine">
+
+                <div class="artTypeButtonContainer">
+                    <button class="artTypeButton" type="button" onclick="showType('sticker',this)">stickers</button>
+                    <button class=" artTypeButton" type="button" onclick="showType('nft',this)">NFT art</button>
+                    <button class="artTypeButton" type="button" onclick="showType('other',this)">other</button>
+                </div>
+
+
+                <script>function showType(type, element) {
+                        var sticker, nft, other, shown;
+                        sticker = document.getElementsByClassName("sticker");
+                        nft = document.getElementsByClassName("nft");
+                        other = document.getElementsByClassName("other");
```

Comment:
Cool to see you adding JavaScript in here! If the JavaScript tends to be more than just a couple lines, move it to a new JS file and link it like we showed in class. So please do that here.